### PR TITLE
Add ZonedDateTime::getDurationTo method

### PR DESCRIPTION
**Pull request by solodkiy**
Just more convenient way to get Duration between two ZonedDateTime

```php
$duration = Duration::between($first->getInstant(), $second->getInstant());
->
$duration = $first->getDurationTo($second);

```

### CODE REVIEW

File: tests/ZonedDateTimeTest.php [hunk 1]
```diff
@@ -841,4 +841,27 @@ private function getTestZonedDateTime(): ZonedDateTime
...
     }
+
+    /**
+     * @dataProvider providerGetDurationTo
+     */
+    public function testGetDurationTo(string $firstDate, string $secondDate, Duration $expectedResult): void
+    {
+        $actualResult = ZonedDateTime::parse($firstDate)->getDurationTo(ZonedDateTime::parse($secondDate));
+
+        $this->assertEquals($expectedResult, $actualResult);
```
Comment (BenMorel) [inline tests/ZonedDateTimeTest.php:852]: Can you please use `assertDurationIs()`?

File: src/ZonedDateTime.php [hunk 2]
```diff
@@ -435,6 +435,15 @@ public function plusDuration(Duration $duration): ZonedDateTime
         return ZonedDateTime::ofInstant($this->instant->plus($duration), $this->timeZone);
     }
 
+    /**
+     * Returns Duration between current and second ZonedDataTime
+     * Duration will be negative if second ZonedDateTime before current
+     */
+    public function getDurationTo(ZonedDateTime $secondZonedDateTime): Duration
```
Comment (BenMorel) [inline src/ZonedDateTime.php:442]: Please use `$that` for consistency with the rest of the codebase!

File: src/ZonedDateTime.php [hunk 3]
```diff
@@ -435,6 +435,15 @@ public function plusDuration(Duration $duration): ZonedDateTime
         return ZonedDateTime::ofInstant($this->instant->plus($duration), $this->timeZone);
     }
 
+    /**
+     * Returns Duration between current and second ZonedDataTime
+     * Duration will be negative if second ZonedDateTime before current
+     */
```
Comment (BenMorel) [inline src/ZonedDateTime.php:441]:
```suggestion
    /**
     * Returns a Duration representing the time elapsed between this ZonedDateTime and the given one.
     * This method will return a negative duration if the given ZonedDateTime is before the current one.
     */
```

**Review by BenMorel — CHANGES_REQUESTED** [review]

**Comment by BenMorel** [comment]
A few nitpicks, otherwise LGTM @solodkiy!

**Comment by solodkiy** [comment]
@BenMorel 
Resolved

**Comment by BenMorel** [comment]
Released in [0.5.1](https://github.com/brick/date-time/releases/tag/0.5.1)!

**Pull request merged**